### PR TITLE
WPF IME - Fixed last hangul letter being duplicated

### LINKED ISSUE #1262: WPF IME Windows 8.1/10
I use the project from official  website and version '41.0.1' ,when I design a wonderful window,I have to set variable ''AllowsTransparency" to "true", “WindowStyle” to "None",I edited the ‘MainWindow.xaml’ ,set variable ''AllowsTransparency" to "true", “WindowStyle” to "None",and run.
the problem:  input tag from html can't use other IME,but character is Ok，how can I solve it?  
Thanks for you

### PR DESCRIPTION
**Pull request by nyjin**
Fixes #1262 

Summary:
- Fixed an issue where the last letter was duplicated when if you press space bar key  or any special key while entering Korean.

Changes:
- Korean characters committed in the IME_ENDCOMPOSITION event are committed once more in the IME_COMPOSITION event. 
So I did not commit in IME_ENDCOMPOSITION event.
      
How Has This Been Tested?
- Tested with sample project.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Updated documentation

**Checklist:**
- [ ] Tested the code(if applicable)
- [ ] Commented my code
- [ ] Changed the documentation(if applicable)
- [ ] New files have a license disclaimer
- [ ] The formatting is consistent with the project (project supports .editorconfig)

### CODE REVIEW

**Comment by nyjin** [comment]
Fixed an issue where the last letter was duplicated when If you press the special key while entering Korean,
https://github.com/cefsharp/CefSharp/issues/1262#issuecomment-465454686

**Comment by amaitland** [comment]
Please add a comment to your code that describes the reason for the branch as it may not be clear to someone viewing the source code at a later date.

File: CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs [hunk 1]
```diff
@@ -253,7 +253,11 @@ public void CancelComposition(IntPtr hwnd)
 
         private void OnImeEndComposition(IntPtr hwnd)
         {
-            owner.GetBrowserHost().ImeFinishComposingText(false);
+            if(languageCodeId != ImeNative.LANG_KOREAN)
```
Comment (amaitland) [inline CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs]: Please add a code comment here explaining the change

Comment (nyjin) [reply CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs]: The comments have been updated.

File: CefSharp3.sln.DotSettings [hunk 2]
```diff
@@ -97,6 +97,9 @@
 	<s:Boolean x:Key="/Default/Environment/SearchAndNavigation/GotoSingleHierarchyItemImmediately/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SearchAndNavigation/GotoSingleUsageImmediately/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SearchAndNavigation/MergeOccurences/@EntryValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
```
Comment (amaitland) [inline CefSharp3.sln.DotSettings]: Revert changes to this file please.

File: CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs [hunk 3]
```diff
@@ -177,31 +177,31 @@ private IntPtr SourceHook(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lParam, re
             switch ((WM)msg)
             {
                 case WM.IME_SETCONTEXT:
-                {
-                    OnImeSetContext(hwnd, (uint)msg, wParam, lParam);
-                    handled = true;
-                    break;
-                }
+                    {
```
Comment (amaitland) [inline CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs]: Revert unnessicary formatting changes please

File: CefSharp3.sln.DotSettings (hunk 2 above)
Comment (nyjin) [reply CefSharp3.sln.DotSettings]: Updated. Sorry to bother you.

File: CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs (hunk 3 above)
Comment (nyjin) [reply CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs]: Updated. Sorry to bother you.

**Pull request merged**